Multisig/UI/App/UIKit/CollectiblesViewController/CollectibleTableViewCell.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  CollectibleTableViewCell.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 02.11.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import UIKit
+
+class CollectibleTableViewCell: UITableViewCell {
+    @IBOutlet weak var cellWrapperView: UIView!
+    @IBOutlet weak var cellImageView: UIImageView!
+    @IBOutlet weak var cellNameLabel: UILabel!
+    @IBOutlet weak var cellDescriptionLabel: UILabel!
+
+    let cornerRadius: CGFloat = 8
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        backgroundColor = .gnoSystemWhite
```

Comment:
The styling code that can be done in XIB, should be done in XIB to reduce the code size

Reply:
ok